src/create/useFormKeyInfosCreate.js

```diff
@@ -0,0 +1,124 @@
+import { useEffect, useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import {
+    validateIsCorrectDate,
+    validateIsNotEmpty,
+    validateShotsIsInRange
+} from '../app/validators.services'
+import loadLocally from '../lib/loadLocally'
+import removeLocally from '../lib/removeLocally'
+import saveLocally from '../lib/saveLocally'
```

Comment:
Perhaps you can also collect the localStorage functions in one file and import them as a bundle.

Reply:
Yeah, that sounds good.